ludwig/schema/hyperopt/__init__.py

```diff
@@ -17,7 +18,14 @@
 class HyperoptConfig(schema_utils.BaseMarshmallowConfig, ABC):
     """Basic hyperopt settings."""
 
-    output_feature: str = "combined"  # TODO: make more restrictive
+    output_feature: str = schema_utils.String(  # TODO: make more restrictive
+        default="combined",
+        description=(
```

Comment:
Nice description.

Note, that there's an aux check that you could potentially re-use, for `hyperopt.output_feature` and `hyperopt.metric` below:

https://github.com/ludwig-ai/ludwig/blob/master/ludwig/config_validation/checks.py#L217-L232